pom.xml

```diff
@@ -32,6 +32,10 @@
     <name>AeroGear UnifiedPush Server</name>
     <url>http://aerogear.org/push</url>
 
+    <organization>
```

Comment:
our parent inherits from Jboss parent, which the correct info:
https://github.com/jboss/jboss-parent-pom/blob/10/pom.xml#L60-L63

No need to override it here